src/UI/class-settings-page.php

```diff
@@ -316,7 +271,7 @@ public function initialize_settings(): void {
 	private function initialize_basic_section(): void {
 		add_settings_section(
 			'basic_settings',
-			__( 'Basic Settings', 'wp-parsely' ),
```

Comment:
Removed because now tabs already shows this and having title also seems redundant.